tests/platform_tests/test_link_down.py

```diff
@@ -158,20 +164,33 @@ def test_link_down_on_sup_reboot(duthosts, localhost, enum_supervisor_dut_hostna
     # RP doesn't have any interfaces, check all LCs' interfaces
     link_status_on_all_LC(duthosts, localhost, fanouts_and_ports, up=False)
 
-    time.sleep(MAX_TIME_TO_REBOOT)
+    # Wait for ssh port to open up on the DUT
+    wait_for_startup(duthost, localhost, 0, MAX_TIME_TO_REBOOT)
 
     dut_uptime = duthost.get_up_time()
     logger.info('DUT {} up since {}'.format(hostname, dut_uptime))
     rebooted = float(dut_uptime_before.strftime("%s")) != float(dut_uptime.strftime("%s"))
     assert rebooted, "Device {} did not reboot".format(hostname)
 
+    # Verify that the links are all LCs are up
+    check_interfaces_and_services_all_LCs(duthosts, conn_graph_facts, xcvr_skip_list)
+
+    # Also make sure fanout hosts' links are up
+    link_status_on_all_LC(duthosts, localhost, fanouts_and_ports)
+
 
 def test_link_status_on_host_reboot(duthosts, localhost, enum_frontend_dut_hostname, 
                                     duts_running_config_facts, conn_graph_facts, 
                                     fanouthosts, xcvr_skip_list, tbinfo):
+    global MAX_TIME_TO_REBOOT
```

Comment:
can remove this

Reply:
Removing the 'global' would imply we have MAX_TIME_TO_REBOOT a local variable within the test, and it doesn't change the value of MAX_TIME_TO_REBOOT that is defined at the top of the test. Thus,  in the other methods like link_status_on_host that are using it, the MAX_TIME_TO_REBOOT is still 120, and not the one that is set in the tests.

Reply:
my point is not removing `global`.
can we do `global MAX_TIME_TO_REBOOT  = 120` outside of any test, at current location on line 7.
and remove ` global MAX_TIME_TO_REBOOT` inside each test

Reply:
Ok - will add another fixture to set this based on ptl_reboot_ctrl values in the next commit